### ISSUE TITLE
"alfred_dvb” cannot be opened because the developer cannot be verified

### ISSUE THREAD

**Issue by TheAlexLichter** [kiliankoe/alfred_dvb#18]
## Error
macOS Catalina will throw the following error when using the workflow:

> “alfred_dvb” cannot be opened because the developer cannot be verified.	
>  macOS cannot verify that this app is free from malware.

## My current workaround

1. Open Alfred
2. Select the workflow
3. Open in Finder
3. Control-click the *alfred_dvb* (unix extension)
3. Choose Open from the shortcut menu
3. Click Open

This will save it as exception

**Comment by kiliankoe** [comment]
Yeah, this is super unfortunate. I'd probably not want to push through the notarization dance with this, need to figure out if that's the only way around this :/
